src/language/branches.md

```diff
@@ -179,12 +184,14 @@ token = switch (charCode) {
 Exhaustiveness checking is a feature that reports a compile-time
 error if it's possible for a value to enter a switch but not match any of the cases.
 
+<?code-excerpt "language/lib/control_flow/branches.dart (exh-bool)"?>
 ```dart
 bool? b = false;
```

Comment:
Also the analyzer can tell this variable isn't null, making the switch exhaustive. Perhaps adjust to call a method like `nullableBool` or something?

Or just move the variable outside of the docregion and name it `nullableBool`?